website/docs/error-reference.md

```diff
@@ -553,6 +553,13 @@ list.each do |elem|
 end
 ```
 
+## 5056
+
+Generated method signatures have been removed from Sorbet.
```

Comment:
```suggestion
The `generated` annotation has been deprecated from Sorbet.
```

Reply:
Suggesting this because it took me a while to figure out why Sorbet was removing generated method before I actually think about `.generated`.